src/main/java/net/sf/jabref/gui/preftabs/NetworkTab.java

```diff
@@ -48,31 +58,60 @@ public NetworkTab(JabRefPreferences prefs) {
         defProxyPort = new JTextField();
         defProxyPort.setEnabled(false);
 
+        useProxyAuthentication = new JCheckBox(Localization.lang("Proxy requires authentication"));
+        useProxyAuthentication.setEnabled(false);
+
+        defProxyUsername = new JTextField();
+        defProxyUsername.setEnabled(false);
+        defProxyPassword = new JPasswordField();
+        defProxyPassword.setEnabled(false);
+        lblPasswordWarning = new JLabel(Localization.lang("Attention: Password is stored in plain text!"));
+        lblPasswordWarning.setEnabled(false);
```

Comment:
Ah, ok, you enable it with the other fields. I overlooked this fact. Leave it as it is.